libs/app-shell/src/lib/api/events/messaging.service.spec.ts

```diff
@@ -139,13 +139,13 @@ describe('MessagingService', () => {
                     eventType: EventType.DEFAULT, shared: true, prefix: 'system:'
                 });
                 let expectedMessage = 'none';
-                service.onMessage('system:events', (msg: Message) => {
+                service.subscribe('system:events', (msg: Message) => {
                     expectedMessage = (msg as TextMessage).text;
                 });
 
                 const m = new TextMessage('system:events');
                 m.text = 'app registered';
-                service.sendTo('system:events', m);
+                service.send('system:events', m);
```

Comment:
When you renamed subcription to subscribe, should not we call thsi publish ?

Reply:
I think it should be emit. Send is a kafka word, but 'send a message' sounds better then 'emit message'. Publish could be better. Because it will be clear pub/sub

Reply:
emit is ugly its too functional and not human friendly ;-) We have publishers/ subcribers, so subcribe and publish